templates/about/release-cycle.html

```diff
@@ -578,15 +578,15 @@ <h2 id="ubuntu-openstack-release-cycle">
     </div>
 
     <div class="u-fixed-width">
-      <p>For more information on previous and upcoming Ubuntu OpenStack releases please see the <a href="https://wiki.ubuntu.com/OpenStack/CloudArchive " class="p-link--external ">Ubuntu Cloud Archive wiki page</a>.</p>
+      <p>For more information on supported versions, see <a href="https://ubuntu.com/openstack/docs/supported-versions" class="p-link--external ">Charmed OpenStack documentation</a>.</p>
     </div>
   </section>
 
   <section class="p-strip--light">
     <div class="row">
       <div class="col-8">
-        <h2 id="charmed-kubernetes-release-cycle">Charmed Kubernetes<sup>&reg;</sup> release cycle</h2>
-        <p>The release cycle of the Charmed Kubernetes is tightly synchronised to the release of upstream Kubernetes<sup>&reg;</sup>. The current release and two prior releases are supported, giving an effective support period of nine months, subject to changes in the upstream release cycle.</p>
+        <h2 id="canonical-kubernetes-release-cycle">Canonical Kubernetes<sup>&reg;</sup> release cycle</h2>
```

Comment:
You changed this id, but you didn't update the `href` in line 45, so that link is broken, can you please update that `href`?

Reply:
Is Canonical Kubernetes a registered trademark? You might need to remove that `<sup>&reg;</sup>;` once you confirm that.

Reply:
Updated the href, will ask about the trademark

Reply:
I think hey [already replied](https://docs.google.com/document/d/1-l3B7AoyDoETaL80UhlfEWCVKmMWPdy3zoF5LpsGcYQ/edit?disco=AAAAOf5RNKA)

Reply:
Whoops, had not seen that, sorry! It's gone now :)